src/components/FavoriteButton.tsx

```diff
@@ -0,0 +1,69 @@
+import { useFocusEffect } from '@react-navigation/native'
+import React, { ReactElement } from 'react'
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import styled from 'styled-components/native'
+
+import { StarCircleIconGrey, StarCircleIconGreyFilled } from '../../assets/images'
+import { Document } from '../constants/endpoints'
+import labels from '../constants/labels.json'
+import useLoadAsync from '../hooks/useLoadAsync'
+import AsyncStorage from '../services/AsyncStorage'
+import { reportError } from '../services/sentry'
+
+const Container = styled.View`
+  padding: ${props => `${props.theme.spacings.xs} 0  ${props.theme.spacings.xs} ${props.theme.spacings.sm}`};
+  align-self: center;
+`
+
+const Icon = styled(StarCircleIconGreyFilled)`
+  min-width: ${wp('9%')}px;
+  min-height: ${wp('9%')}px;
+`
+const IconOutline = styled(StarCircleIconGrey)`
+  min-width: ${wp('9%')}px;
+  min-height: ${wp('9%')}px;
+`
+const Button = styled.TouchableOpacity`
+  justify-content: center;
+  align-items: center;
+  shadow-radius: 5px;
```

Comment:
add here shadow-color as for the audio player. i think we should have the same shadow color for all the icons
```js
shadow-color: ${props => props.theme.colors.shadow};
```

Reply:
added shadow